ygot/struct_validation_map.go

```diff
@@ -82,6 +82,44 @@ func structTagToLibPaths(f reflect.StructField, parentPath *gnmiPath, preferShad
 	return mapPaths, nil
 }
 
+// structTagToLibModules takes an input struct field as a reflect.Type, and
+// determines the set of validation library modules that it maps to. Returns
```

Comment:
What does `validation library` mean in this context? Is it schema modules?

Reply:
Fixed to be clearer. I copied over the other docstring and didn't proofread.

Reply:
Thanks!